e2etest/go.mod

```diff
@@ -16,10 +16,10 @@ require (
 	github.com/prometheus/client_golang v1.19.0 // indirect
 	go.uber.org/zap v1.26.0 // indirect
 	golang.org/x/sys v0.18.0 // indirect
-	k8s.io/api v0.30.1
+	k8s.io/api v0.30.3
```

Comment:
why did we need this bump (sorry if I asked already)?

Reply:
probably not, I revert and I added only the changes that this command
```
go get k8s.io/kubectl/pkg/drain@v0.30.1
```
gave me .